### PR TITLE
Add stable instructions for Linux

### LINKED ISSUE #219: Linux: Update to RC, Remove Offline/Online
https://community.openhab.org/t/openhab-2-0-0-release-candidate/20462/3

@kaikreuzer you were tagged in this thread.
I could do the needed changes, maybe @BClark09 want's to continue?

### PR DESCRIPTION
**Pull request by BClark09**
Depends on #207 
Depends on 2.0.0 Release ;)

Closes #219

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

File: installation/linux.md [hunk 1]
```diff
@@ -61,60 +61,57 @@ First, add the openHAB 2 Bintray repository key to your package manager:
...
-As long as openHAB 2 is in its development state you may choose between the latest *Beta* release or a *Snapshot* build.
-The snapshot build is created [almost daily](https://bintray.com/openhab/apt-repo2/openhab2#release), always including the latest changes to the openHAB 2 core and add-ons.
-Decide between two options:
+Then, you can choose between, *Official (Stable)*, *Beta* or *Snapshot* builds:
 
-* **Beta Release**
+* **Official Release**
 
-  Add the **openHAB 2 Beta Repository** to your systems apt sources list (will overwrite the existing `openhab2.list`):
+  Contains a stable release with tested features passed the test of time from the builds below.
```
Comment (ThomDietrich) [inline installation/linux.md]: "test of time" ?

File: installation/linux.md [hunk 2]
```diff
@@ -61,60 +61,57 @@ First, add the openHAB 2 Bintray repository key to your package manager:
...
-  sudo apt-get install openhab2-offline
-  # OR #
-  sudo apt-get install openhab2-online
+  echo 'deb http://dl.bintray.com/openhab/apt-repo2 testing main' | sudo tee /etc/apt/sources.list.d/openhab2.list
   ```
 
 * **Snapshot Release**
 
-  Add the **openHAB 2 Unstable Repository** to your systems apt sources list (will overwrite the existing `openhab2.list`):
+  The snapshot build is created [almost daily](https://bintray.com/openhab/apt-repo2/openhab2#release), and includes the latest changes to the openHAB 2 core and add-ons.
```
Comment (ThomDietrich) [inline installation/linux.md]: Give an additional warning? "Only for testing or if instructed by developers" or something like that

File: installation/linux.md [hunk 3]
```diff
@@ -61,60 +61,57 @@ First, add the openHAB 2 Bintray repository key to your package manager:
...
-  ```shell
-  sudo apt-get install openhab2
-  ```
+```shell
+sudo apt-get install openhab2
+```
 
-  The latest snapshots will download add-ons on request, if you plan on disconnecting your machine from the internet then you will want to also install the add-ons package.
-  Optionally, you can also install the legacy add-ons package, which contains [1.x bindings]({{base}}/addons/bindings.html#legacy-1x-bindings) for compatibility (useful if you're [coming from openHAB 1.x]({{base}}/tutorials/migration.html) for example):
+When you choose to install an add-on, openHAB will download it from the internet on request, if you plan on disconnecting your machine from the internet then you will want to also install the add-ons package.
```
Comment (ThomDietrich) [inline installation/linux.md]: Better break down to two sentences?

File: installation/linux.md (hunk 1 above)
Comment (BClark09) [inline installation/linux.md]: Sorry, is this a British Idiom?  Could probably remove that last bit.

File: installation/linux.md (hunk 2 above)
Comment (BClark09) [inline installation/linux.md]: Sure!

File: installation/linux.md (hunk 3 above)
Comment (BClark09) [inline installation/linux.md]: Yep, you shall have your line-break ;)

**Comment by BClark09** [comment]
Responded to your comments, an collapsible item for them would be great. How do we add it in markdown format?

**Comment by ThomDietrich** [comment]
You don't ;) Check the bottom:
https://raw.githubusercontent.com/ThomDietrich/openhab-docs/7f372271837b994d58721986fdfee39f1c702fac/addons/uis/apps/android.md

File: installation/linux.md (hunk 1 above)
Comment (ThomDietrich) [inline installation/linux.md]: I don't know mate. I'm german and never heard that expression.

File: installation/linux.md (hunk 3 above)
Comment (ThomDietrich) [inline installation/linux.md]: xD that will haunt me forever now

**Comment by BClark09** [comment]
Okay, removed the references to online and offline, and also added the information about changing versions, just in case.

**Comment by ThomDietrich** [comment]
Hey Ben, looks great! Will merge as soon as the package is available.

Regarding the collapsible: Don't bother with it. I will do a few style changes in the next days and look into this as well.

**Comment by BClark09** [comment]
Thanks @ThomDietrich, I'll hold on the collapsible. Looking forward to seeing what you come up with.

**Comment by ThomDietrich** [comment]
@BClark09 I've clarified the meaning of "Legacy 1.x bindings" after having the discussion in the other thread.

**Comment by BClark09** [comment]
That's cool, makes more sense this way now.

**Comment by BClark09** [comment]
Hi @ThomDietrich Did you ever look into the collapsible class? I'm having a few issues with syntax highlighting within them:

```
**Meeting the Requirements:**
As a first step, please verify, that your system meets the [prerequisites](index.html#prerequisites).

<div><ul class="collapsible"><li><div class="collapsible-header">

APT Based

</div><div class="collapsible-body"><span markdown="1">

A repository providing the latest Oracle Java 8 revision (above "101") is being maintained by the [Webupd8 Team](https://launchpad.net/~webupd8team/+archive/ubuntu/java).
Follow the provided guides for either a [repository based](http://www.webupd8.org/2012/09/install-oracle-java-8-in-ubuntu-via-ppa.html) or a [PPA based](http://www.webupd8.org/2014/03/how-to-install-oracle-java$
In short these are the commands to execute step-by-step on most systems:

```shell
echo "deb http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main" | sudo tee /etc/apt/sources.list.d/webupd8team-java.list
echo "deb-src http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main" | sudo tee -a /etc/apt/sources.list.d/webupd8team-java.list
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys EEA14886
sudo apt-get update
sudo apt-get install oracle-java8-installer
sudo apt-get install oracle-java8-set-default
\```

</span></div></li><li><div class="collapsible-header">

RPM Based

</div><div class="collapsible-body"><span markdown="1">

#Hello

</span></div></li></ul></div>

```

Produces:
image

**Comment by ThomDietrich** [comment]
Hey, sadly I didn't have the time to look into that yet -.- Just as you, I *should* be working on my PhD.

I can do some testing myself. Will be back soon.

**Comment by BClark09** [comment]
Haha, thanks @ThomDietrich! I hope the PhD is going well anyhow!

I found that the code ticks won't work within <span> tags so simply having:

```
</div><div class="collapsible-body" markdown="1">
```

Creates:

image

It looks slightly weird, but a lot better than a massive block of text!

**Comment by ThomDietrich** [comment]
Definitely looks better.
You know, this could be related to the style changes I was planning since long. #248  
Fixing the syntax highlighting problems might also deal with your problem...
